src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,39 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj) {
+            return true;
+        }
+
+        if (obj == null) {
+            return false;
+        }
+        if (!(obj instanceof Rectangle)) {
```

Comment:
should we use here getClass() or instanceof ? 
and why

Reply:
We should use getClass() because we can get troubles when this class get child class